products/ol7/profiles/e8.profile

```diff
@@ -130,6 +130,7 @@ selections:
   - sshd_disable_empty_passwords
   - sshd_disable_user_known_hosts
   - sshd_enable_strictmodes
+  - sshd_strong_macs=default
```

Comment:
The CI fail looks legit, you will have to update the profile stability tests.

Reply:
Should be ok in 7db8bb6